nevergrad/functions/olympus/core.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Based on https://github.com/aspuru-guzik-group/olympus
+
+import matplotlib.pyplot as plt
+import numpy as np
+from nevergrad.parametrization import parameter as p
+from ..base import ExperimentFunction
+import olympus
+
+
+class Olympus(ExperimentFunction):
+    def __init__(self, kind : str, dimension: int = 10) -> None:
+        self.kind = kind
+        self.param_dim = dimension
```

Comment:
This is correct but does not inform the optimization algorithm that the objective function is noisy, I would do something like

parametrization = p.Array(shape=(dimension,))
parametrization.function.deterministic = False

(@jrapin , big boss of parametrization stuff, might know more than i do :-) )